demo/dlb-init.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -eu
+
+enable_and_configure_vfs() {
+  devpath=$1
+  num_vfs_per_pf=$2
+  # enable sriov
+  sriov_numvfs_path="$devpath/sriov_numvfs"
+  if ! test -w "$sriov_numvfs_path"; then
+    echo "error: $sriov_numvfs_path is not found or not writable. Check if dlb driver module is loaded."
```

Comment:
if you call exit here you don't need 'else', which would make your code less nested and more readable.